models/record.go

```diff
@@ -0,0 +1,93 @@
+package models
```

Comment:
@postables once this PR lands should I have another crack at extracting the database stuff into RTradeLtd/database? a bit worried about the deviation between the packages in Temporal growing too large to perform a non-trivial merge upstream

Reply:
@bobheadxi  yes that sounds like a great idea! I imagine now that we got rid of that stinking utility that was causing issues it should be easy to do.